pkg/composableschemadsl/compiler/importer.go

```diff
@@ -26,63 +25,18 @@ type CircularImportError struct {
 	filePath string
 }
 
-func importFile(importContext importContext) (*CompiledSchema, error) {
```

Comment:
Lots of logic moved out of here and into `translateImport` because we're no longer recursing here.